specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2022-07-01-preview/deviceupdate.json

```diff
@@ -1604,10 +1609,10 @@
           "Device and deployment management"
         ],
         "description": "Get the best available update for a device class subgroup and a count of how many devices need this update.",
-        "operationId": "DeviceManagement_ListBestUpdatesForDeviceClassSubgroup",
+        "operationId": "DeviceManagement_GetBestUpdatesForDeviceClassSubgroup",
```

Comment:
Please be sure the change on operationId is intended for all languages (not only for .NET).

Reply:
No, we don't want to make any breaking change in this service version (since it was already reviewed and published). At this point we are not trying to fix swagger but to release the corresponding SDK client libraries.

Reply:
I agree with @weidongxu-microsoft , pageable have to be called `List`, C# codegen has a regexp automatic rule to replace them with `Get`. By renaming them `Get`, it makes other languages bad and do not take benefit of the C# rename.
If for some reason the rename do not happen in DPG C#, then let's do a transform rule for now.

cc @AlexanderSher @annelo-msft

Reply:
Ok, this operation is not pageable, forget my comment :p